doc/tor.md

```diff
@@ -29,8 +34,13 @@ outgoing connections, but more is possible.
 
 	-onlynet=onion  Make outgoing connections only to .onion addresses. Incoming
 	                connections are not affected by this option. This option can be
-	                specified multiple times to allow multiple network types, e.g.
-	                ipv4, ipv6, or onion.
+	                specified multiple times to define a set of general allowed network
+	                types, e.g. ipv4, ipv6 or onion. If you use this option
+	                with other values than onion you can *not* disable onion connections.
+	                Onion outbound connections will be enabled when you use the -proxy
+	                or the -onion option.
+	                Please use -noonion or -onion=0 if you want to be sure to have no
+	                outbound onion connections over the default proxy or your defined proxy..
 
```

Comment:
"set of general allowed network types"  seems wordy and doesn't appear to add anything over the original text.

If onlynet=ipv4 still makes onion connections, that sounds like a bug to me.  Is there any discussion about this?

Reply:
a *bug* , yup and the bug has even deeper onions and layers of head scratching, some fix in #19358 review more than welcome.

Reply:
@Saibato your use of double tilde for mdash (I think that's what you mean?) makes it look like you strike out your own words, which I don't think you intended.

Reply:
@ysangkok 
>  I don't think you intended

;-) was a try to high light  without adjectives,  while not saying even more harsh words about what happened all those years (was kind of a privacy strike through), since overall slow reaction and some disbelieve, my say it nice clock has almost reached zero  , like it, that u noticed. :+1: